packages/components/Avatar/package.json

```diff
@@ -36,18 +36,31 @@
     "@fluentui-react-native/interactive-hooks": "^0.18.1",
     "@fluentui-react-native/theme-tokens": "^0.19.0",
     "@fluentui-react-native/tokens": "^0.16.0",
-    "@fluentui-react-native/use-styling": "^0.9.0",
-    "react-native-svg": "^12.3.0"
```

Comment:
Interesting this moved to devDependencies?

Reply:
Yeah, I'm not sure why. It's clearly used as part of the Avatar component. I can add it back.